optimizely/3540700460-sem-campaign.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Experiment ID: 3540700460
+ * Description: SEM campaign effectiveness
+ * Mana URL: https://mana.mozilla.org/wiki/display/FIREFOX/%5Bbug+1205745%5D+SEM+campaign+effectiveness
+ */
+
+// Regexp used for targeting
+function snippetRegexp() {
+    'use strict';
+    return /^.*www\.mozilla\.org\/.+\/firefox\/new\/.+utm_source=google.+utm_medium=paidsearch.+utm_campaign=sem2015Q3.*$/;
```

Comment:
Since this test is en-US only, could simplify the regex a bit by specifying locale.